### PR TITLE
Fix geometry demo

### PR DESCRIPTION
**Pull request by EdwardMoyse**
Get the geometry demo working properly again.

The problem was the code to draw parametrised geometry was lost in the move to angular. So I re-implemented this, and added two new functions:
- buildGeometryFromParameters to EventDisplay
- addGeometryFromParameters to ThreeManager

Thanks to @9inpachi for the help!

### CODE REVIEW

File: packages/phoenix-event-display/src/three/index.ts [hunk 1]
```diff
@@ -579,4 +583,52 @@ export class ThreeManager {
   public getObjectByName(objectName: string): Object3D {
     return this.getSceneManager().getScene().getObjectByName(objectName);
   }
+
+  public addGeometryFromParameters(parameters: any): void {
```
Comment (9inpachi) [inline packages/phoenix-event-display/src/three/index.ts:587]: Can you quickly add docs to this function? Or may be do it in another PR or commit.
We should have the documentation to 100% for the event display. :)

**Review by 9inpachi — CHANGES_REQUESTED** [review]

File: packages/phoenix-event-display/src/three/index.ts (hunk 1 above)
Comment (EdwardMoyse) [reply packages/phoenix-event-display/src/three/index.ts:590]: Good point! Thanks! 

I actually want to change `any` to specify the required parameters... but the main thing was to get this working before the presentation next week.
